cluster/local/microk8s.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1090
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${scriptdir}/../.. build.vars)
+
+MICROK8S_REGISTRY="localhost:32000"
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+# Use microk8s registry add-on
+MICROK8S_IMAGE="${MICROK8S_REGISTRY}/${PROJECT_NAME}:master"
+DEFAULT_NAMESPACE="crossplane-system"
+
+function copy_image_to_cluster() {
+    local build_image=$1
+    local final_image=$2
+    docker tag "${build_image}" "${final_image}" && docker push "${final_image}"
+    echo "Tagged image: ${final_image}"
+}
+
+# Deletes pods with application prefix. Namespace is expected as the first argument
+function delete_pods() {
+    for pod in $(kubectl get pods -n "$2" -l "app=$1" --no-headers -o custom-columns=NAME:.metadata.name); do
+        kubectl delete pod "$pod" -n "$2"
+    done
+}
+
+# current kubectl context == minikube, returns boolean
+function check_context() {
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "minikube" ]; then
+        return 0
+    fi
+
+    return 1
+}
+
+case "${1:-}" in
+  up)
+    microk8s.start
+
+    microk8s.enable ingress
+
+    microk8s.enable registry
+
+    kubectl apply -f ${scriptdir}/helm-rbac.yaml
+    ${HELM} init --service-account tiller
+
+    copy_image_to_cluster ${BUILD_IMAGE} ${MICROK8S_IMAGE}
```

Comment:
When I first ran `microk8s.sh up`, port `32000` was not yet ready.  It took a minute or so to become available,